.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: go
 go:
-    - 1.6.x
```

Comment:
it was a context type mismatch that broke grpc 

Go 1.7 moved golang.org/x/net/context to the standard library "context".